lms/djangoapps/courseware/tests/test_crowdsource_hinter.py

```diff
@@ -0,0 +1,244 @@
+"""
+Test scenarios for the crowdsource hinter xblock.
+"""
+import json
+import itertools
+import StringIO
+from ddt import ddt, data
+from copy import deepcopy
+
+from django.conf import settings
+from django.core.urlresolvers import reverse
+from django.test.utils import override_settings
+
+from xmodule.modulestore.tests.factories import CourseFactory, ItemFactory
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase, mixed_store_config
+from xmodule.modulestore.tests.factories import XMODULE_FACTORY_LOCK
+
+from courseware.tests.helpers import LoginEnrollmentTestCase
+from courseware.tests.factories import GlobalStaffFactory
+
+from lms.djangoapps.lms_xblock.runtime import quote_slashes
+
+class TestCrowdsourceHinter(ModuleStoreTestCase, LoginEnrollmentTestCase):
+
+    STUDENTS = [
+        {'email': 'view@test.com', 'password': 'foo'},
+        {'email': 'view2@test.com', 'password': 'foo'}
+    ]
+    XBLOCK_NAMES = ['crowdsource_hinter']
+
+    XMODULE_FACTORY_LOCK.enable()
```

Comment:
The comment on XModuleFactoryLock says, "Users of XModuleFactory (or its subclasses) should only call XModuleFactoryLock.enable after ensuring that a) the modulestore will be cleaned up, and b) that XModuleFactoryLock.disable will be called."    I don't see where .disable will be called here.  This seems wrong.